src/NamedDims.jl

```diff
@@ -7,7 +7,7 @@ using Pkg
 using Requires
 using Statistics
 
-export NamedDimsArray, dim, refine_names, rename, unname, dimnames
+export NamedDimsArray, dim, rename, unname, dimnames
```

Comment:
Perhaps it should still be exported though, else you won't get the `@deprecate`d function?

Reply:
yeah, that's right. I believe it is exported still via the `true` arg in `@deprecate refine_names NamedDimsArray true`

Reply:
Oh I didn't know that, sorry. All good then!

Reply:
yeah, no worries. magical bools are magical. and it made me go double-check which was good :)